src/material_pool.hpp

```diff
@@ -132,7 +137,7 @@ namespace wr
 				float m_roughness;
 				float m_emissive_multiplier;
 				float m_is_double_sided;
-				float m_use_alpha_constant;
+				float m_use_alpha_masking;
```

Comment:
I think this is because c++ sees bool as 1 byte, hlsl sees a bool as 4 bytes. Then again we're packing the values in an uint32.. Perhaps thats the confusion?

Reply:
Why is m_is_double_sided a float?
I just renamed the variable and refactoring the material pool is not the aim of this branch